BrewMesModulesParent/common/src/main/java/com/brewmes/common/services/IMachineService.java

```diff
@@ -0,0 +1,65 @@
+package com.brewmes.common.services;
+
+import com.brewmes.common.entities.Connection;
+import com.brewmes.common.util.Command;
+import com.brewmes.common.util.Products;
+
+import java.util.List;
+
+public interface IMachineService {
+
+    /**
+     * Takes a command and sends it to the desired machine
+     * @param command The command that should be sent to the machine
+     * @param machineId The ID of the machine that should receive the command
+     */
+    void controlMachine(Command command, String machineId);
+
+    /**
+     * Sets the desired values for a production
+     * @param speed The desired speed that the production should run at
+     * @param beerType The desired beer type of the production
+     * @param batchSize The desired amount of beer to be produced
+     * @param machineId The ID of the desired machine the values should be set on
+     */
+    void setVariables(double speed, Products beerType, int batchSize, String machineId);
+
+    /**
+     * Sets up the desired subscriptions to a machines values.
+     * @param machineId The ID of the machine desired for the subscription
+     */
+    void subscribeToMachineValues(String machineId);
```

Comment:
```suggestion
    boolean subscribeToMachineValues(String machineId);
```
Should probably return a response if the subscription was successfully set up or not.